calculator/operations.py

```diff
@@ -1,19 +1,31 @@
 
 def add(*args):
     # your implementation here
-    pass
+    total = args[0]
+    for i in args[1:]:
+        total += i
+    return total
 
 def subtract(*args):
     # your implementation here
-    pass
+    total = args[0]
+    for i in args[1:]:
+        total -= i
+    return total
 
 def multiply(*args):
     # your implementation here
-    pass
+    total = args[0]
+    for i in args[1:]:
+        total *= i
+    return total
 
 def divide(*args):
     # your implementation here
-    pass
+    total = args[0]
+    for i in args[1:]:
+        total /= float(i)
+    return total
```

Comment:
I was tempted to use `float()` but I was thinking: if my numbers are 6 / 2, I want to return 3 not 3.0

So I used:
```python
from __future__ import division
```
Now the Python2 division acts exactly like in Python3. :)